IFC4x3/Properties/u/UndercontrolEquipmentsID_0uZsrOMH5AMBQLhPS3ukXv/Documentation.md

```diff
@@ -0,0 +1 @@
+The identity of the equipment controlled by the power control system.
```

Comment:
This is a bit vague as a description. How are these supposed to be formatted? GUIDs , IFC identifiers, STEP line IDs?

Reply:
This property is removed: https://github.com/bSI-InfraRoom/IFC-Specification/pull/191/commits/d82022836002a1eb860d4f69d1bdd8ad4615ef74